### ISSUE TITLE
Issues with processing FLEXPART footprint

### ISSUE THREAD

**Issue by gareth-j** [openghg/openghg#623]
I've just tried processing `ECO_5_mAGL_EUROPE_202101.nc` using `standardise_footprint`, doing

```python
res = standardise_footprint(filepath=fp_path, site="ECO", domain="EUROPE", 
                                               model="FLEXPART", inlet="5magl")
```

I first get the error

```python
ValueError: Continuous data with no gaps is expected but no time period can be inferred. 
Run with continous=False to remove this constraint.
```

So I do as I'm told and add `continuous=False` but get another error due to a time units issue.

```python
res = standardise_footprint(filepath=fp_path, site="ECO", domain="EUROPE", model="FLEXPART", inlet="5magl", continuous=False)

File ~/Documents/Devel/openghg/openghg/store/base/_datasource.py:133, in <dictcomp>(.0)
    129 data = data.sortby(time_coord)
    131 # Use a dictionary keyed with the daterange covered by each segment of data
    132 # Group by year
--> 133 new_data = {
    134     self.get_representative_daterange_str(year_data, period=period): year_data
    135     for _, year_data in data.groupby(f"{time_coord}.year")
    136 }
    138 # Ensure daterange strings are independent and do not overlap each other
    139 # (this can occur due to representative date strings)
    140 new_data = self._clip_daterange_label(new_data)

File ~/Documents/Devel/openghg/openghg/store/base/_datasource.py:353, in get_representative_daterange_str(self, dataset, period)
    349 start_date, end_date = self.get_dataset_daterange(dataset)
    351 # If period is defined add this to the end date
    352 # This ensure start-end range includes time period covered by data
--> 353 if period is not None:
    354     period_td = relative_time_offset(period=period)
    355     end_date = (
    356         end_date + period_td - Timedelta(seconds=1)
    357     )  # Subtract 1 second to make this exclusive end.

File ~/Documents/Devel/openghg/openghg/util/_time.py:833, in relative_time_offset(value, unit, period)
    831     time_delta = DateOffset(**{unit: value})
    832 else:
--> 833     time_delta = time_offset(value, unit)
    835 return time_delta

File ~/Documents/Devel/openghg/openghg/util/_time.py:792, in time_offset(value, unit, period)
    787 if unit in ("weeks", "months", "years"):
    788     raise ValueError(
    789         "Unable to calculate time offset with unit of {unit}. Try relative_time_offset() function instead"
    790     )
--> 792 time_delta = Timedelta(value, unit)
    794 return time_delta

File ~/Documents/Devel/env/openenv/lib/python3.9/site-packages/pandas/_libs/tslibs/timedeltas.pyx:1694, in pandas._libs.tslibs.timedeltas.Timedelta.__new__()

File ~/Documents/Devel/env/openenv/lib/python3.9/site-packages/pandas/_libs/tslibs/timedeltas.pyx:721, in pandas._libs.tslibs.timedeltas.parse_timedelta_unit()

ValueError: invalid unit abbreviation: varies
```

I haven't had a chance to dive into this any deeper but will soon. The NetCDF doesn't have many attributes stored within it so we may need to provide more information if we're processing these files.

**Comment by rt17603** [comment]
I had a quick look and this seems to be related to a floating point difference in the hourly frequency of the "time" value within the  data. This is meaning we can't infer the frequency and having the knock on effect shown.

I think it's a reasonable expectation that we can handle this level of variation so I'll treat this as  a bug and take a look at fixing this bug so we can check this footprint file can be successfully standardised.

**Comment by gareth-j** [comment]
I'm having a look at this now.

**Comment by gareth-j** [comment]
This is fixed with #634
